pyanaconda/modules/storage/devicetree/handler.py

```diff
@@ -82,16 +82,20 @@ def teardown_device(self, device_name):
             msg = "Failed to tear down {}: {}".format(device_name, str(e))
             raise DeviceSetupError(msg) from None
 
-    def mount_device(self, device_name, mount_point):
+    def mount_device(self, device_name, mount_point, options=""):
         """Mount a filesystem on the device.
 
         :param device_name: a name of the device
         :param mount_point: a path to the mount point
+        :param options: options used for device mount, could be an empty string
         :raise: MountFilesystemError if mount fails
         """
         device = self._get_device(device_name)
         try:
-            device.format.mount(mountpoint=mount_point)
+            if options:
+                device.format.mount(mountpoint=mount_point, options=options)
```

Comment:
Please, replace the branches with the following code:
`device.format.mount(mountpoint=mount_point, options=options or None)`